examples/Training/matlab/OMEROTables.m

```diff
@@ -0,0 +1,97 @@
+% Copyright (C) 2017 University of Dundee & Open Microscopy Environment.
+% All rights reserved.
+%
+% This program is free software; you can redistribute it and/or modify
+% it under the terms of the GNU General Public License as published by
+% the Free Software Foundation; either version 2 of the License, or
+% (at your option) any later version.
+%
+% This program is distributed in the hope that it will be useful,
+% but WITHOUT ANY WARRANTY; without even the implied warranty of
+% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+% GNU General Public License f4or more details.
+%
+% You should have received a copy of the GNU General Public License along
+% with this program; if not, write to the Free Software Foundation, Inc.,
+% 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+% ROIs
+
+% To learn about the model see  http://www.ome-xml.org/wiki/ROI/2010-04.
```

Comment:
Additionally in the case of tables, a link to the OME Model will be of little use as this is largely an OMERO concept